verification/curator-service/ui/src/components/landing-page/LandingPage.test.tsx

```diff
@@ -165,8 +165,11 @@ describe('<SignUpForm />', () => {
             />,
         );
 
-        userEvent.type(screen.getByLabelText('Email'), 'test@email.com');
-        userEvent.type(screen.getByLabelText(/Confirm Email/), 'xxx@email.com');
+        userEvent.type(screen.getByLabelText('Email *'), 'test@email.com');
+        userEvent.type(
+            screen.getByLabelText(/Confirm Email */),
```

Comment:
in a regex, `*` means "any number of the previous character" so this test isn't actually testing for the presence of the asterisk (you need `\*`).